### DISCUSSION TITLE
Login Feature

### DISCUSSION THREAD

**Discussion by ExtremeMachine12** [tanmaypardeshi/CSC-510-Project3-PopcornPicks#3] [General]
I am almost finished with the login feature. It will be finished by tonight, and I will merge request to the master.

**Comment by ExtremeMachine12** [comment]
I've finished the feature. It includes being able to sign up, log in, and log out. There is also light error detection. For example, users can't register with emails that have already been used to register. Obviously, users can only login with existing accounts with the correct credentials.

**Reply by tanmaypardeshi** [reply]
Thanks a lot Sam! I will thoroughly review your PR tonight and leave my comments on it.

**Comment by tanmaypardeshi** [comment]
I have left some comments on your PR. Everything looks good to me. Just some minor comments. Thanks!

**Comment by tanmaypardeshi** [comment]
Thanks for the fixes Sam! I have approved and merged your code into the main brannch. I am closing this discussion now.
